docs_website/docs/developer_guide/run_db_migration.md

```diff
@@ -4,12 +4,27 @@ title: DB Migration
 sidebar_label: DB Migration
 ---
 
-To make changes to the SQLAlchemy database model definitions, you need to run the following migrations:
+## Adding a new DB migration
+
+First, make changes to the SQLAlchemy database model definitions under `querybook/server/models/`,
+then you can run the following code to generate the migration file:
 
 ```sh
-# Edit querybook/alembic.ini sqlalchemy.url to point to the database
+# Edit querybook/alembic.ini sqlalchemy.url to point to the database you want to change
 # go into the docker terminal
 cd querybook
 PYTHONPATH=server alembic revision --autogenerate -m "<>commit message<>"
+```
+
+Remember to always update the package version when making migration changes.
+
+## Applying DB migrations
+
+After pulling the latest changes, you can apply the migrations by running the following:
+
+```sh
+# Edit querybook/alembic.ini sqlalchemy.url to point to the database you want to change
+# go into the docker terminal
```

Comment:
will any of the container work or it has to be the web container?
could you just give an example command for it? like `docker exec -it querybook_web bash`